libs/game/controller.ts

```diff
@@ -142,19 +144,35 @@ namespace controller {
             if (!this._pressed) return;
             this._pressedElasped += dtms;
 
+            const delay = this.repeatDelay === undefined ? defaultRepeatDelay : this.repeatDelay;
+            const interval = this.repeatInterval === undefined ? defaultRepeatInterval : this.repeatInterval;
+
             // inital delay
-            if (this._pressedElasped < this.repeatDelay)
+            if (this._pressedElasped < delay)
                 return;
 
             // repeat count for this step
-            const count = Math.floor((this._pressedElasped - this.repeatDelay) / this.repeatInterval);
+            const count = Math.floor((this._pressedElasped - delay - interval) / interval);
             if (count != this._repeatCount) {
                 this.raiseButtonRepeat();
                 this._repeatCount = count;
             }
         }
     }
 
+    /**
+     * Set default values for repeat button event delay and interval
```

Comment:
Configures the timing of the `on button repeat` event for all of the controller buttons